Makefile

```diff
@@ -107,7 +107,7 @@ docker-buildbot-master:
 	$(DOCKERBUILD) -t buildbot/buildbot-master:master master
 
 $(VENV_NAME):
-	virtualenv -p $(VENV_PY_VERSION) --no-site-packages $(VENV_NAME)
```

Comment:
Do you know the reason why this does not work by chance? In my experience this leads to various issues depending on what packages and what versions of them are installed on the system.

Reply:
I documented what I did in the PR description.

for your specific question, according to:

https://stackoverflow.com/questions/60783788/virtualenv-virtualenvwrapper-virtualenv-error-unrecognized-arguments-no-sit

> --no-site-packages is the default for virtualenv (and has been for like 5 years?) you can remove export VIRTUALENVWRAPPER_VIRTUALENV_ARGS='--no-site-packages' from your .bashrc
> it appears in virtualenv>=20 that this option was removed

was news to me as well :-/

Reply:
Thanks, it all makes sense now.